zcash_primitives/src/transaction/mod.rs

```diff
@@ -58,7 +58,7 @@ const V5_VERSION_GROUP_ID: u32 = 0x26A7270A;
 #[cfg(zcash_unstable = "nu6")] /* TODO nu7 */
 const V7_TX_VERSION: u32 = 7;
 #[cfg(zcash_unstable = "nu6")] /* TODO nu7 */
-const V7_VERSION_GROUP_ID: u32 = 0x26A7270C; // TODO ???
+const V7_VERSION_GROUP_ID: u32 = 0x124A69F8; // TODO ???
```

Comment:
If this is final - please remove TODO comment

Reply:
I picked a value (at random) in `zcash-test-vectors`, and set this to match that value. I'm not sure if there is a value suggested by Zcash yet. That's why I left the TODO comment.

Reply:
Ah, ok, then keep it